Microsoft.Azure.Cosmos/src/Query/v3Query/CosmosQueryClientCore.cs

```diff
@@ -63,17 +63,23 @@ internal class CosmosQueryClientCore : CosmosQueryClient
                 trace,
                 cancellationToken);
 
-            string effectivePartitionKeyString = null;
+            List<Range<string>> effectivePartitionKeyRange = null;
```

Comment:
nit: Shouldn't this be simply:

```suggestion
            Range<string> effectivePartitionKeyRange = null;
```